browser/main/lib/dataApi/attachmentManagement.js

```diff
@@ -624,6 +624,52 @@ function deleteAttachmentsNotPresentInNote (markdownContent, storageKey, noteKey
   }
 }
 
+function getAttachments (markdownContent, storageKey, noteKey) {
```

Comment:
The name of the function is missleading.. it only returns certain attachments, won't it?

Reply:
Well it get attachment paths for 3 type of attachments:
- Existing attachments
- Existing attachments that are in use
- Existing attachments that aren't in use

What name should I put it? How about `getAttachmentsPath`?

Reply:
hm.. don't know.. if you stay with the name, can you add javadoc explaining what it returns?